content/swagger/resources/products_uuid/definitions/product_uuid.yaml

```diff
@@ -146,10 +146,10 @@ properties:
     x-from-version: '5.0'
   completenesses:
     type: array
-    description: Product completenesses for each channel/locale combination (only available on SaaS platforms, and when the "with_completenesses" query parameter is set to "true")
+    description: Product completenesses for each channel/locale combination (only available since 7.0, and when the "with_completenesses" query parameter is set to "true")
```

Comment:
```suggestion
    description: Product completenesses for each channel/locale combination (only available since the 7.0 version, and when the "with_completenesses" query parameter is set to "true")
```

Reply:
6.0 or 7.0 ... not sure about when the completeness has been added to the PIM API